src/components/ProgressIndicator/index.js

```diff
@@ -37,12 +37,21 @@ export default class ProgressIndicator extends Component {
     }
 
     render() {
-        const { style, className, children, currentStepName, onClick } = this.props;
+        const { style, className, variant, children, currentStepName, onClick } = this.props;
+        const numbersMap = React.Children.toArray(children).reduce((map, child, index) => {
```

Comment:
here be careful with this since steps can be not the child, for example when wrapping with RenderIf